src/test/components/MenuButtons.test.js

```diff
@@ -278,6 +285,31 @@ describe('app/MenuButtons', function() {
       expect(queryPreferenceCheckbox()).toBeFalsy();
     });
 
+    it('can publish and revert', async () => {
```

Comment:
Thought: lol, this is such a short test for what it took to get there. It's a shame our tests weren't more robust here.